packages/dd-trace/src/config.js

```diff
@@ -163,6 +164,36 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_TRACEPARENT_ENABLED,
       false
     )
+    const DD_TRACE_PROPAGATION_STYLE_INJECT = coalesce(
+      options.tracePropagationStyle && options.tracePropagationStyle.inject,
```

Comment:
Should we also support setting a value directly on `tracePropagationStyle`? For example when someone uses b3 I'd expect that they want everything to use b3.

Reply:
Maybe? The spec doesn't include that though. If we want to do that we should probably make sure it's consistent across languages and probably have a `DD_TRACE_PROPAGATION_STYLE` env var to correspond to that. 🤔

Reply:
Specs don't usually include programmatic options at all since these are vastly different across languages.

Reply:
It could always be added later though, this was just a thought.

Reply:
Perhaps we should skip this for now and only do something about it later if we get users confused/complaining about having two different configs? Probably best to avoid proliferation of configs unless we have a compelling reason for it.

Reply:
Looks like a decision was made on what the shared config should look like, so I've implemented it.